new/LIT6420MJMarriageCana.xml

```diff
@@ -0,0 +1,77 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6420MJMarriageCana" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of Jesus: The marriage at Cana</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="RL"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="RL" when="2021-03-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6420MJMarriageCana" passive="LIT2382Taamme"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+              <bibl>
+                <ptr target="bm:RompayArras1975Miracles"/><citedRange unit="item">31</citedRange>
+              </bibl>
+            </div>
+            <div type="edition">
+              <note>The following division into textparts and excerpts are taken from <ref type="mss" corresp="EMIP02392"/>,
+                  without subscription and supplications.</note>
+                  <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>
+                      በእንተ፡ አመ፡ ረሰዮ፡ ለማይ፡ ወይነ።<gap reason="omitted"/> ወሀሎ፡ ፩ብእሲ፡ በሀገረ፡ ባቲላስ፡ እንተ፡ ይእቲ፡ ምድረ፡ ቃና፡ ዘገሊላ፡ ከብካበ፡ ለዘመደ፡ አትናኤል።
```

Comment:
not ናትናኤል?

Reply:
The ms has አትናኤል. Should we mark this with with 'sic'
![EMIP02392_Weiner357_026](https://user-images.githubusercontent.com/39619238/121157509-884b5580-c841-11eb-9cbb-a7df88ffbe79.jpg)

Reply:
No, I think it can stay as it is, I was just wondering whether it was a typo